<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listPlates">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>열림</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>화면종료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newregisterButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>신규등록</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
